src/select.coffee

```diff
@@ -44,7 +44,12 @@ Ember.Widgets.SelectOptionView = Ember.ListItemView.extend
     # propagate the click event
     # if the dropdown is expanded and we select something, don't propagate
     if @get('controller.showDropdown')
-      @get('controller').send 'hideDropdown'
+      # Keep showing dropdown if it is multi select component
+      unless @get('controller.layoutName') is 'multi-select'
```

Comment:
this is pretty bad, we should definitely find a way to override `click` in `multi-select` so that `select` doesn't have to have logic for `multi-select`.

I'd recommend refactoring lines 40-42 into a `selectContent` function, calling that in the `click` handler for both `select` and `multi-select`, and then having each component handle its own logic for hiding the dropdown. You could then remove the `@_super` call from the `multi-select` click handler.